portfolio/src/main/webapp/index.html

```diff
@@ -16,40 +17,45 @@ <h1>George Zheng</h1>
     </div>
     <div id="background">
       <p> I have worked on a variety of projects spanning several academic disciplines (check out some selected works featured below!). In my free time, I enjoy playing video games (most recently, League of Legends and Realm of the Mad God), watching anime, and napping. 
```

Comment:
nit: remove the space between ```<p>``` and the text.

Reply:
Done.